# Service to activate EGM
bool mode
int64 timeout
---
int64 ret
string msg
